packages/strapi-plugin-content-type-builder/admin/src/containers/App/tests/actions.test.js

```diff
@@ -397,6 +413,38 @@ describe('App actions', () => {
           ],
         },
       ];
+      const groups = {
+        data: [
+          {
+            uid: 'ingredients',
+            name: 'Ingredients',
+            source: null,
+            schema: {
+              connection: 'default',
+              collectionName: 'ingredients',
+              description: 'Little description',
+              attributes: {
+                name: {
+                  type: 'string',
+                  required: true,
+                },
+                quantity: {
+                  type: 'float',
+                  required: true,
+                },
+                // A discuter du nouveau format pour les relations avec vous
```

Comment:
Probably best to have comments in english:
`To discuss the new format for relations with you`

Reply:
This comment needs to be removed